testsuite/performance/stress-test.sh

```diff
@@ -84,23 +122,31 @@ case "${algorithm}" in
 
     bisection)
 
+        echo "  STRESS_TEST_UPS_LOWER_BOUND: $lowerBound users per second"
+        echo "  STRESS_TEST_UPS_UPPER_BOUND: $upperBound users per second"
+        echo "  STRESS_TEST_UPS_RESOLUTION: $resolution users per second"
+
         for (( i=0; i < $maxIterations; i++)); do
 
-            intervalSize=`echo "$highPoint - $lowPoint" | bc`
-            usersPerSec=`echo "$lowPoint + $intervalSize * 0.5" | bc`
-            if [[ `echo "$intervalSize < $tolerance" | bc`  == 1 ]]; then echo "INFO: intervalSize < tolerance. Stopping the loop."; break; fi
-            if [[ `echo "$intervalSize < 0" | bc`           == 1 ]]; then echo "ERROR: Invalid state: lowPoint > highPoint. Stopping the loop."; exit 1; fi
-            ITERATION_INFO="L: $lowPoint    H: $highPoint   intervalSize: $intervalSize   tolerance: $tolerance"
+            intervalSize=`echo "$upperBound - $lowerBound" | bc`
+            usersPerSec=`echo "$lowerBound + $intervalSize * 0.5" | bc`
```

Comment:
By default, `bc` peforms integer computation:
```
$ bc <<<'4/3'
1
```
`scale` needs to be set for non-integer results:
$ bc <<<'scale=2 ; 4/3'
1.33
```